resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/Main.scala

```diff
@@ -16,43 +16,47 @@ object Main
-        val shouldRunCheckInitBuckets =
-          Opts.flag("checkInitBuckets", "checks that init buckets for deleted runtimes are deleted").orFalse
+        val shouldCheckAll = Opts.flag("all", "run all checks").orFalse
+        val shouldCheckDeletedRuntimes = Opts.flag("checkDeletedRuntimes", "check all deleted runtimes").orFalse
+        val shouldCheckErroredRuntimes = Opts.flag("checkErroredRuntimes", "check all errored runtimes").orFalse
+        val shouldCheckStoppedRuntimes = Opts.flag("checkStoppedRuntimes", "check all stopped runtimes").orFalse
         val shouldRunCheckDeletedKubernetesClusters =
           Opts.flag("checkDeletedKubernetesClusters", "check all deleted or errored kubernetes clusters").orFalse
         val shouldRunCheckDeletedNodepools =
           Opts.flag("checkDeletedNodepools", "check all deleted or errored nodepools").orFalse
+        val shouldCheckDeletedDisks = Opts.flag("checkDeletedDisks", "check all deleted disks").orFalse
+        val shouldCheckInitBuckets =
+          Opts.flag("checkInitBuckets", "checks that init buckets for deleted runtimes are deleted").orFalse
 
         (enableDryRun,
-         shouldRunAll,
-         shouldRunCheckDeletedRuntimes,
-         shouldRunCheckErroredRuntimes,
-         shouldRunCheckDeletedDisks,
-         shouldRunCheckInitBuckets,
+         shouldCheckAll,
+         shouldCheckDeletedRuntimes,
+         shouldCheckErroredRuntimes,
+         shouldCheckStoppedRuntimes,
          shouldRunCheckDeletedKubernetesClusters,
-         shouldRunCheckDeletedNodepools).mapN {
+         shouldRunCheckDeletedNodepools,
+         shouldCheckDeletedDisks,
+         shouldCheckInitBuckets).mapN {
           (dryRun,
-           runAll,
+           checkAll,
            shouldCheckDeletedRuntimes,
-           shouldRunCheckErroredRuntimes,
-           shouldRunCheckDeletedDisks,
-           shouldRunCheckInitBuckets,
-           shouldRunCheckDeletedKubernetesClusters,
-           shouldRunCheckDeletedNodepools) =>
+           shouldCheckErroredRuntimes,
+           shouldCheckStoppedRuntimes,
+           shouldCheckDeletedKubernetesClusters,
+           shouldCheckDeletedNodepools,
+           shouldCheckDeletedDisks,
+           shouldCheckInitBuckets) =>
             ResourceValidator
               .run[IO](
-                isDryRun = dryRun,
-                shouldRunAll = runAll,
-                shouldRunCheckDeletedRuntimes = shouldCheckDeletedRuntimes,
-                shouldRunCheckErroredRuntimes = shouldRunCheckErroredRuntimes,
-                shouldRunCheckDeletedDisks = shouldRunCheckDeletedDisks,
-                shouldRunCheckInitBuckets = shouldRunCheckInitBuckets,
-                shouldRunCheckDeletedKubernetesCluster = shouldRunCheckDeletedKubernetesClusters,
-                shouldRunCheckDeletedNodepool = shouldRunCheckDeletedNodepools
+                dryRun,
```

Comment:
we added the named param so that it's less easy to get the param order wrong....can you add it back

Reply:
I see. Sure.